notebooks/concepts/contour_overlay.ipynb

```diff
@@ -0,0 +1,146 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Contour Overlay "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports \n",
+    "* _numpy_ for array math\n",
+    "* _astropy.io_ for reading and writing FITS cubes and images\n",
+    "* _matplotlib.pyplot_ for plotting spectra and images."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy.io import fits\n",
+    "from astropy.visualization import SqrtStretch\n",
+    "from astropy.visualization.mpl_normalize import ImageNormalize\n",
+    "import numpy as np\n",
+    "\n",
+    "import matplotlib\n",
+    "from matplotlib import pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Read in data cube\n",
+    "Read in NIRSpec IFU data cube from Box.   This particular example is a simulated quasar + host galaxy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BoxPath='https://data.science.stsci.edu/redirect/JWST/jwst-data_analysis_tools/cube_fitting/Q3D_20200407/'\n",
+    "fname='Q3D_NRS_491_s3d.fits'\n",
+    "filename=BoxPath+fname\n",
+    "hdul1=fits.open(filename)\n",
+    "hdul1.info()\n",
+    "\n",
+    "with fits.open(filename, memmap=False) as hdulist:\n",
+    "    sci = hdulist[\"SCI\"].data\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sum over spectral axis\n",
+    "\n",
+    "This could be any derivative data product (emission line flux, equivalent width, velocity map, etc...)"
```

Comment:
```suggestion
    "This could be any derivative data product of the same shape as the spatial part of the cube (emission line flux, equivalent width, velocity map, etc...). Here we are doing one of the simplest operations - a sum, for illustrative purposes."
```